apps/web/app/s/[surveyId]/page.tsx

```diff
@@ -105,12 +105,12 @@ const Page = async (props: LinkSurveyPageProps) => {
   }
 
   // verify email: Check if the survey requires email verification
-  let emailVerificationStatus: string = "";
+  let emailVerificationStatus = "";
   let verifiedEmail: string | undefined = undefined;
 
   if (survey.isVerifyEmailEnabled) {
     const token =
-      searchParams && Object.keys(searchParams).length !== 0 && searchParams.hasOwnProperty("verify")
+      Object.keys(searchParams).length !== 0 && Object.prototype.hasOwnProperty.call(searchParams, "verify")
```

Comment:
can we just use `searchParams.verify` here? 
```js
 const token = searchParams.verify 
```
This way its only dependent on just the property and the checks below this would also work